package.json

```diff
@@ -78,6 +78,7 @@
     "setupTestFrameworkScriptFile": "./test-setup.js"
   },
   "dependencies": {
+    "css-spring": "^4.1.0",
```

Comment:
Are you animating something?

Reply:
Oh no! Accidentally left that in when trying to demo some things for Zac